src/main/proto/wfa/measurement/reporting/v1alpha/event_group.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.reporting.v1alpha;
+
+import "google/api/resource.proto";
+
+option java_package = "org.wfanet.measurement.reporting.v1alpha";
+option java_multiple_files = true;
+
+// A grouping of events defined by a `DataProvider`. For example, a single
+// campaign or creative defined in a publisher's ad system.
+message EventGroup {
+  option (google.api.resource) = {
+    type: "reporting.halo-cmm.org/EventGroup"
+    pattern: "measurementConsumers/{measurement_consumer}/dataProviders/{data_provider}/eventGroups/{event_group}"
+  };
+
+  // Resource name.
+  string name = 1;
+
+  // Resource name of the `DataProvider` associated with this `EventGroup`.
+  // Immutable.
+  string data_provider = 2 [(google.api.resource_reference).type =
```

Comment:
I added `data_provider` as your example in previous discussion, but now I can't remember the reason. Is `data_provider` in the `EventGroup` necessary for listing event groups?